functions/src/index.ts

```diff
@@ -108,5 +108,6 @@ export const api = functions.https.onRequest(app_secure);
 export const challenge = functions.https.onRequest(app_open);
 
 // firestore triggers
+export const build_vanity_link = vanityFunctions.build_vanity_link;
 export const create_vanity_link = vanityFunctions.create_vanity_link;
-export const email_discord_mapper = hacktoberfestFunctions.mapper;
+//export const email_discord_mapper = hacktoberfestFunctions.mapper;
```

Comment:
Is this now deprecated? Why is it commented out?

Reply:
Will uncomment for now. These functions need to be archived with documentation at a later date.